nemo/collections/cv/modules/video_preprocessing.py

```diff
@@ -0,0 +1,109 @@
+import torch
+import torchvision
+from torch import nn
+
+from nemo.collections.cv.modules import Permute
+from nemo.core.classes import NeuralModule, typecheck
+
+
+class VideoPreprocessor(NeuralModule):
+
+    """ Lip Reading Video Pre-processing
+
+    args:
+        grayscale: convert images to grayscale
+        normalize: normalize videos
+        resize: resize videos
+        resize_size: output image size for resize
+        norm_mean: normalize mean
+        norm_std: normalize std
+    
+    """
+
+    def __init__(self, grayscale, normalize, resize, resize_size, norm_mean, norm_std):
+        super().__init__()
+
+        # Params
+        self.grayscale = grayscale
+        self.normalize = normalize
+        self.resize = resize
+        self.resize_size = resize_size
+        self.norm_mean = norm_mean
+        self.norm_std = norm_std
+
+        self.transforms = nn.ModuleList()
+
+        # Convert float32 [0:255] -> [0:1]
+        self.transforms.append(torchvision.transforms.ConvertImageDtype(dtype=torch.float32))
```

Comment:
This function can probably be done in normal pytorch without depending on torchvision

Reply:
I kept torchvision.transforms.ConvertImageDtype for now